audio/opus-tools/Portfile

```diff
@@ -5,9 +5,9 @@ PortGroup       muniversal 1.0
 PortGroup       compiler_blacklist_versions 1.0
 
 name            opus-tools
-version         0.1.10
+version         0.2
 categories      audio
-license         BSD
+license         BSD GPL-2
```

Comment:
Why is the license being changed?  The website does not make mention of a change to GPL-2.  What portions are GPL-2?

Reply:
opusdec and opusenc are BSD licensed and opusinfo is GPL-2 licensed.
See: https://github.com/xiph/opus-tools/blob/master/COPYING#L29-L30
It didn't change upstream; the MacPorts port was incorrectly marked.

Reply:
Thanks.  I see that now lower down in https://opus-codec.org/license as well.

Let's skip installing opusinfo, split it out to a subport, or make it available via variant similar to how ffmpeg was done.  I'd prefer to not force GPL unexpectedly on folks.